wp-content/themes/haruoshi_origin/src/scss/single-works.scss

```
@@ -0,0 +1,34 @@
+@import "variables";
+@import "mixin";
+
+.container {
+	inset: 0;
+	margin: $header-height auto 60px;
+	max-width: $contents-max-width;
+	width: calc(100% - 20px);
+	&__title {
+		font-size: 24px;
+		font-weight: bold;
+	}
+	&__date {
+		display: block;
+		margin: 20px 0;
+	}
+	&__thumbnail {
+		max-width: $contents-max-width;
+		width: calc(100% - 20px);
+		height: auto; // IE対応のために指定
+	}
+	&__contents {
+		margin: 60px 0;
+	}
+}
+
+.back {
+	margin-bottom: 240px;
+	text-align: center;
+	&__link {
+		display: inline-block;
+		border-bottom: solid 1px $main-color;
```

```suggestion
		text-decoration: underline;
```
下に線引くって意味では一緒なんだけどやりたいことはこっちな気がする

---

done https://github.com/oooshima/origin_wordpress/pull/28/commits/9deb70afcb61ae60ce18931099cff8f31796e61a